dev-server/documents/html/raven.mustache

```diff
@@ -0,0 +1,727 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html;charset=iso-8859-1" />
+    <title>The Project Gutenberg eBook of The Raven, by Edgar Allan Poe.</title>
+    <style type="text/css">
+      body {
+        background-color: black;
```

Comment:
I think the interesting aspect of this example should be called out more loudly, eg. via some comments near the top of the file.